src/modules/esl-utils/dom/script.ts

```diff
@@ -1,23 +1,30 @@
+export interface loadScriptOptions {
+  crossorigin?: boolean | 'anonymous' | 'use-credentials' | '';
+  integrity?: string;
+  referrerpolicy?: ReferrerPolicy;
+}
+
 /**
- * Creates new async script tag by id and src
+ * Creates new script element with specified attributes
```

Comment:
```suggestion
 * Creates new script element with specified attributes
 * Note: as the script will be added dynamically, it will be treated as async by the browser
```